=== modified file 'CMakeLists.txt'
--- CMakeLists.txt	2013-11-18 16:04:23 +0000
+++ CMakeLists.txt	2013-11-22 17:49:59 +0000
@@ -270,9 +270,18 @@
 find_package(Cairo 1.8.1 QUIET)
 check_find_package_result(CAIRO_FOUND "Cairo")
 
-# Download boost and possibly build parts of it
-#################################################
-include( download_boost )
+if( APPLE )
+    ######################
+    # Find Boost library #
+    ######################
+    find_package(Boost REQUIRED COMPONENTS date_time filesystem iostreams locale program_options regex system thread )
+    check_find_package_result(Boost_FOUND "boost")
+else()
+    #################################################
+    # Download boost and possibly build parts of it #
+    #################################################
+    include( download_boost )
+endif()
 
 
 ##########################

