=== modified file 'CMakeModules/download_boost.cmake'
--- CMakeModules/download_boost.cmake	2013-11-12 12:44:30 +0000
+++ CMakeModules/download_boost.cmake	2013-11-13 15:46:28 +0000
@@ -108,11 +108,45 @@
 
     set( bootstrap ./bootstrap.sh --with-libraries=${libs_csv} )
     # pass to *both* C and C++ compilers
-    set( PIC_STUFF "cflags=${PIC_FLAG}" )
+    if( APPLE )
+        set( PIC_STUFF "" )
+    else()
+        set( PIC_STUFF "cflags=${PIC_FLAG}" )
+    endif()
     set( BOOST_INCLUDE "${BOOST_ROOT}/include" )
     unset( b2_libs )
 endif()
 
+if( APPLE )
+    ExternalProject_Add( boost
+        PREFIX          "${PREFIX}"
+        DOWNLOAD_DIR    "${DOWNLOAD_DIR}"
+        URL             http://downloads.sourceforge.net/project/boost/boost/${BOOST_RELEASE}/boost_${BOOST_VERS}.tar.bz2
+        URL_MD5         ${BOOST_MD5}
+
+        # The patch command executes with the working directory set to <SOURCE_DIR>
+        # PATCH_COMMAND   bzr patch -p0 "${PROJECT_SOURCE_DIR}/patches/boost.patch"
+        PATCH_COMMAND   bzr revert
+        # PATCH_COMMAND continuation (any *_COMMAND here can be continued with COMMAND):
+            COMMAND     bzr patch -p0 "${PROJECT_SOURCE_DIR}/patches/boost_minkowski.patch"
+            COMMAND     bzr patch -p0 "${PROJECT_SOURCE_DIR}/patches/boost_cstdint.patch"
+
+        # [Mis-]use this step to erase all the boost headers and libraries before
+        # replacing them below.
+        UPDATE_COMMAND  ${CMAKE_COMMAND} -E remove_directory "${BOOST_ROOT}"
+
+        CONFIGURE_COMMAND ""
+
+        # remove then re-copy into the include/boost directory during next two steps:
+        BUILD_COMMAND   ${CMAKE_COMMAND} -E remove_directory ${BOOST_ROOT}
+        INSTALL_COMMAND ${CMAKE_COMMAND} -E copy_directory "${headers_src}" "${BOOST_ROOT}/include/boost"
+        )
+
+    # Until my find_package() support is done for my boost.
+    # set( Boost_INCLUDE_DIR  "${BOOST_ROOT}/include" CACHE FILEPATH "Boost include directory" )
+
+else()
+
 ExternalProject_Add( boost
     PREFIX          "${PREFIX}"
     DOWNLOAD_DIR    "${DOWNLOAD_DIR}"
@@ -148,6 +182,7 @@
 
     INSTALL_COMMAND ""
     )
+endif()
 
 if( MINGW )
     execute_process( COMMAND ${CMAKE_C_COMPILER} -dumpversion

